src/webgpu/api/operation/device/info.spec.ts

```diff
@@ -0,0 +1,34 @@
+export const description = `
+Tests GPUDevice.adapterInfo member.
+`;
+
+import { Fixture } from '../../../../common/framework/fixture.js';
+import { makeTestGroup } from '../../../../common/framework/test_group.js';
+import { getGPU } from '../../../../common/util/navigator_gpu.js';
+import { assert, objectEquals } from '../../../../common/util/util.js';
+
+export const g = makeTestGroup(Fixture);
+
+g.test('device_adapter_info')
```

Comment:
nit: test name is redundant with filename; maybe `'matches_adapter'`

or if merged into `adapter,info`, `device_matches_adapter`

Reply:
Done